pkg/endpointmanager/conntrack.go

```diff
@@ -77,14 +93,26 @@ func RunGC(e *endpoint.Endpoint, isLocal, isIPv6 bool, filter *ctmap.GCFilter) {
 			"count":         deleted,
 		}).Debug("Deleted filtered entries from map")
 	}
+
+	return resets
 }
 
 // EnableConntrackGC enables the connection tracking garbage collection.
 func EnableConntrackGC(ipv4, ipv6 bool) {
+	v6ResetMetrics, err := metrics.DatapathManagementErrors.GetMetricWith(labelIPv6CTDumpReset)
+	if err != nil {
+		log.WithError(err).Info("Failed to fetch IPv6 CT Dump gauge; disabling metric")
+	}
+	v4ResetMetrics, err := metrics.DatapathManagementErrors.GetMetricWith(labelIPv4CTDumpReset)
```

Comment:
There are some functions that panic if the lookup fails but allow you to chain the `.Set` at the end directly, if you wanted to avoid the nil check below and have all the metrics stuff be one line.